src/lib/index/wb_cache.cpp

```diff
@@ -141,41 +155,43 @@ void IndexWBCache::read_buf(bnodeid_t id, BtreeNodePtr& node, node_initializer_t
 
 std::tuple< bool, bool > IndexWBCache::create_chain(IndexBufferPtr& second, IndexBufferPtr& third, CPContext* cp_ctx) {
     bool second_copied{false}, third_copied{false};
-
+    auto chain = second;
+    auto old_third = third;
     if (!second->is_clean()) {
```

Comment:
this line not needed right? Given copy_buffer already did the clean state check.

Reply:
This is to check if we need to even call copy_buffer and update the return value of second. copy_buffer is also used in refresh_node, so.